### DISCUSSION TITLE
删除视频弹窗删不掉左下角的这个

### DISCUSSION THREAD

**Discussion by KuiPer-LTW** [the1812/Bilibili-Evolved#4703] [功能建议 / Ideas]
我自己用ADGuard屏蔽了展开的弹窗，但是缩小的弹窗过滤规则搞不明白，选择屏蔽没有用，然后这个组件也是一样的问题

![image](https://github.com/the1812/Bilibili-Evolved/assets/103661658/2354ea5d-88ce-40db-a46d-c821da8cfd4d)

**Comment by KuiPer-LTW** [comment]
如果选择展开弹窗让它屏蔽，用手机看的时候就又要关，就很恶心

**Comment by the1812** [comment]
视频链接发出来, 这个算新的弹窗了

**Reply by KuiPer-LTW** [reply]
https://www.bilibili.com/video/BV1WA4m1L7xa/

**Reply by the1812** [reply]
好像没有啊
<img src="https://github.com/the1812/Bilibili-Evolved/assets/26504152/9434057c-c01f-49ca-ba7c-cae9e9fda1b5" height="400">

**Reply by KuiPer-LTW** [reply]
暂停没有，播放就有了，而且你有没有先关掉组件把那个选项缩小，我说了如果是展开的就会完全屏蔽，但是展开在手机上也会同步展开，sbB站
